Bottom: 938b7b3c8fbeb5be67b89fe70f57bd9531bf1282
Top:    8964793ffc031d1754f0da46fef51db4d3443455
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-26 23:38:52 +0100

Refresh of add-soldermask-layers

---

diff --git a/src/const.h b/src/const.h
index 42a04db..193ebc8 100644
--- a/src/const.h
+++ b/src/const.h
@@ -48,6 +48,8 @@
  */
 #define BOTTOM_SILK_LAYER       0
 #define TOP_SILK_LAYER          1
+#define BOTTOM_SOLDERMASK_LAYER 2
+#define TOP_SOLDERMASK_LAYER    3
 
 /* ---------------------------------------------------------------------------
  * misc constants
